queries/models/monitoramento/staging/schema.yml

```
@@ -89,4 +89,39 @@ models:
+      - name: indicador_temperatura_descartada
+        description: "Indicador se mais de 50% dos registros de temperatura foram descartados"
+      - name: indicio_falha
+        description: "Indicador de indício de falha no sistema de ar condicionado"
+      - name: quantidade_dia_falha_operacional
+        description: "Quantidade consecutiva de dias com falha operacional do ar condicionado"
+      - name: datetime_ultima_atualizacao
+        description: "{{ doc('datetime_ultima_atualizacao') }}"
+      - name: versao
+        description: "{{ doc('versao') }}"
+      - name: id_execucao_dbt
+        description: "{{ doc('id_execucao_dbt') }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)